packages/yoastseo/src/languageProcessing/helpers/sanitize/quotes.js

```diff
@@ -1,11 +1,14 @@
+const SINGLE_QUOTES_ARRAY = [ "'", "‘", "’", "‛", "`", "‹", "›" ];
+const SINGLE_QUOTES_REGEX = /['‘’‛`‹›]/g;
```

Comment:
The DRY'er option would be to create the regex using `SINGLE_QUOTES_ARRAY`. But this is nitpicking 😅